awx/main/management/commands/run_callback_receiver.py

```diff
@@ -147,7 +147,7 @@ def callback_worker(self, queue_actual, idx):
                     from pprint import pformat
                     logger.info('Body: {}'.format(
                         highlight(pformat(body, width=160), PythonLexer(), Terminal256Formatter(style='friendly'))
-                    ))
+                    )[:1024 * 4])
```

Comment:
This truncates the debug output for the callback receiver so we don't print gobs and gobs of text for large stdout content.

Reply:
Though some people might depend on this output? But I'm for it.

Reply:
This only happens when `settings.DEBUG is True` - more of a helper for development so that you can see event data as it gets consumed.